httpcore5-h2/src/main/java/org/apache/hc/core5/http2/impl/nio/bootstrap/H2RequesterBootstrap.java

```diff
@@ -303,6 +305,16 @@ public final H2RequesterBootstrap setUriPatternType(final UriPatternType uriPatt
         return this;
     }
 
+    /**
```

Comment:
@psmarko This is a good start. Please add `@since 5.4`  tag to this method. There should also be a similar setter added to `H2MultiplexingRequesterBootstrap` and `H2ServerBootstrap`